### DISCUSSION TITLE
Octant Docker Extension

### DISCUSSION THREAD

**Discussion by ivan-curkovic** [docker/extension-ideas#158] [Ideas]
octant
Visualize your Kubernetes workloads
Octant is an open source developer-centric web interface for Kubernetes that lets you inspect a Kubernetes cluster and its applications.
[octant.dev](https://octant.dev/)

**Comment by AlanGreene** [comment]
fyi VMWare have ended active development of Octant as of Jan 19, 2023: https://github.com/vmware-archive/octant

**Reply by ivan-curkovic** [reply]
Hi @AlanGreene, Thanks for letting us know.
